Makefile

```diff
@@ -1,18 +1,23 @@
 .PHONY: lint test all docs unittest
 
 SOURCES := speechmatics/ setup.py
+VERSION ?= $(shell cat VERSION).dev0
 
 all: lint test docs
 
 lint:
 	pylint $(SOURCES)
 	flake8 $(SOURCES)
+	pycodestyle $(SOURCES)
```

Comment:
doesn't flake8 also wrap around pycodestyle?

Reply:
I thought it did, yet, all of the line length issues were caught by `pycodestyle` when I ran it separately, flake8 said "all clear".

Reply:
hmm, funny, i suppose it doesn't hurt running it separately